cmd/maestro/server/pulse_server.go

```diff
@@ -0,0 +1,120 @@
+package server
```

Comment:
Can you add integration test for `pulseServer`?

Reply:
added integration test for pulse server.